### PR TITLE
Increase time periods for promQL expressions

### PR DESCRIPTION
**Pull request by paulfantom**
We need to ensure enough datapoints being present when doing calculations over time (like `rate()`). Since we are scraping data every 30s it means we need to have a period of at least 2m and in case of metrics coming from cAdvisor library - at least 5m. 

Metrics coming from cAdvisor need more time as they contain timeseries that are respected by prometheus. This, in turn, can affect calculations.

/cc @brancz @s-urbaniak

### CODE REVIEW

File: jsonnet/rules.jsonnet [hunk 1]
```diff
@@ -75,15 +75,15 @@
             record: 'cluster:capacity_memory_bytes:sum',
           },
           {
-            expr: 'sum(1 - rate(node_cpu_seconds_total{mode="idle"}[1m]) * on(namespace, pod) group_left(node) node_namespace_pod:kube_pod_info:{})',
+            expr: 'sum(1 - rate(node_cpu_seconds_total{mode="idle"}[2m]) * on(namespace, pod) group_left(node) node_namespace_pod:kube_pod_info:{})',
```
Comment (s-urbaniak) [inline jsonnet/rules.jsonnet]: let's go with `[5m]` as @brancz suggested out of band.

Comment (paulfantom) [reply jsonnet/rules.jsonnet]: This is not a cadvisor metric, but node_exporter one, so no need for 5m.

**Pull request merged**